docs/source/index.rst

```diff
@@ -72,9 +72,7 @@ Getting Started
    :maxdepth: 1
    :caption: Getting Started
 
-   tutorials/audio_io_tutorial
-   tutorials/streaming_api_tutorial
-   tutorials/streaming_api2_tutorial
+   tutorials.io
```

Comment:
this is currently not showing up under the "Getting Started" section on the rendered docs
<img width="300" alt="Screen Shot 2022-09-23 at 1 30 42 PM" src="https://user-images.githubusercontent.com/16568633/191960574-eecd7b7c-b37d-4f58-b8e1-744a49649c26.png">

Reply:
oops, the last minute filename change caused this. fixing...